appveyor.yml

```diff
@@ -28,7 +28,6 @@ init:
 install:
     - mamba env create -f environment.yml
     - conda activate cadquery
-    - pip install git+https://github.com/CadQuery/OCP-stubs.git@7.6.3
```

Comment:
The stubs really aren't needed anymore?

Reply:
They already specified in the environment file

Reply:
Ok, thanks.